scripts/actions/reset_default_app

```diff
@@ -0,0 +1,259 @@
+#!/bin/bash
```

Comment:
(somewhat related to the comment about actions.toml ... if we really want to have this sort of mecanism to reset the app conf, we should probably go for like an "upgrade --force" which should achieve the same effect rather than yet-another-unmaintainable-duplication-of-every-logic-bit)